circe.el

```diff
@@ -1949,7 +1956,12 @@ setting the variable through network options."
   (with-circe-server-buffer
     (let* ((target-name (irc-isupport--case-fold (circe-server-process)
                                                  target))
-           (chat-buffer (generate-new-buffer target))
+           (chat-buffer (generate-new-buffer
+                         (lui-format 'circe-format-buffer-name
+                                     `(target
+                                       ,target
+                                       network
+                                       ,(with-circe-server-buffer circe-network)))))
```

Comment:
The `with-circe-server-buffer` part shouldn't be necessary as that's already used earlier in the code.

Reply:
Sorry, I didn't see that, fixed.